Source/WebCore/html/CanvasBase.cpp

```diff
@@ -73,6 +74,7 @@ CanvasBase::~CanvasBase()
     ASSERT(m_didNotifyObserversCanvasDestroyed);
     ASSERT(m_observers.isEmptyIgnoringNullReferences());
     ASSERT(!m_imageBuffer);
+    m_canvasNoiseHashSalt = std::nullopt;
```

Comment:
Nit - is explicitly clearing `m_canvasNoiseHashSalt` necessary in the destructor?